<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Plots</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="tsplot_layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
